Assets/Plugins/iOS/Shopify/BuyTests/Models/Models.swift

```diff
@@ -116,6 +117,10 @@ struct Models {
             shippingMethod: createShippingMethod())
     }
     
+    static func createSummaryItem() -> PKPaymentSummaryItem {
+        return PKPaymentSummaryItem(label: "SubTotal", amount: NSDecimalNumber.init(value: 1.00))
```

Comment:
Nit: Don't need to explicitly call out `.init` - `NSDecimalNumber(value: 1.00)` works!